kickstarts/base.ks.erb

```diff
@@ -103,9 +103,19 @@ EOF
 # Let's rebuild the ramfs with with base scsi drivers we need
 kversion=$(rpm -q kernel --qf '%{version}-%{release}.%{arch}\n')
 ramfsfile="/boot/initramfs-$kversion.img"
-/sbin/dracut --force --add-drivers "mptbase mptscsih mptspi<%= " hv_storvsc hid_hyperv hv_netvsc hv_vmbus" if @target == "hyperv" || @target == "azure" %>" $ramfsfile $kversion
-
-<%= render_partial "post/aws-ec2" if @target == "aws-ec2" %>
+<% case @target
+   when "vsphere", "vagrant" %>
+drivers="mptbase mptscsih mptspi"
+<% when "azure", "hyperv" %>
+drivers="hv_storvsc hid_hyperv hv_netvsc hv_vmbus"
+<% when "ec2" %>
+drivers="xen-blkfront xen-netfront"
+<% else %>
+drivers=""
+<% end %>
+test -n "$drivers" && /sbin/dracut --force --add-drivers "$drivers" $ramfsfile $kversion
```

Comment:
Updated PR so that we continue to install the LSI logic drivers on all platforms, so that we don't introduce big changes shortly before the Euwe release.

Will add submit a PR to master to remove them.